### DISCUSSION TITLE
[QUESTION/REQUEST]: Release Checksums?

### DISCUSSION THREAD

**Discussion by sibeov** [typst/typst#4559] [Ideas]
# Request
Hello there.

Would it be possible to include a SHA256 (or similar) for the downloadable release binaries? 
Would be neat the ensure the integrity of the downloads.

## Example
```powershell
$downloaded_file_hash = (Get-FileHash -Path .\path\to\downloaded\typst\binary -Algorithm SHA256).Hash
$compare_chechsum_from_github_release_page = "7987AD872229879797ADDD7987987ADD8" #Some random number
if ( $downloaded_file_hash -eq  $compare_chechsum_from_github_release_page) {
  Write-Output "Hurray"
}
else {
  Write-Output "Noooooooooooooooooooooooooooo.."
}
```

Kind Regards
Enthusiastic typst discoverer

**Comment by runner12434** [comment]
I think you can add it as an issue. Signing the binaries via PGP is even better, imo.
